build.sh

```diff
@@ -21,7 +19,7 @@ for php_version in "${php_versions[@]}"; do
         -t "visiblevc/wordpress:latest" \
         -t "visiblevc/wordpress:latest-php${php_version}" \
         -t "visiblevc/wordpress:$npm_package_version-php${php_version}" \
-        "$dockerfile_dir"
+        .
```

Comment:
Not needed since running using npm will ensure that the `$PWD` is the root of the repo.